lib/peripheral.dart

```diff
@@ -28,32 +47,57 @@ class Peripheral {
           refreshGatt: refreshGatt,
           timeout: timeout);
 
+  /// Observe connection state
+  ///
+  /// Returns stream of [PeripheralConnectionState]
   Stream<PeripheralConnectionState> observeConnectionState(
           {bool emitCurrentValue = false, bool completeOnDisconnect = false}) =>
       _manager.observePeripheralConnectionState(
           identifier, emitCurrentValue, completeOnDisconnect);
 
+  /// Check connection state of the Device.
   Future<bool> isConnected() => _manager.isPeripheralConnected(identifier);
 
+  /// Disconnects from Device if it's connected or cancels pending connection.
   Future<void> disconnectOrCancelConnection() =>
       _manager.disconnectOrCancelPeripheralConnection(identifier);
 
+  /// Discovers all Services, Characteristics and Descriptors for Device.
+  ///
+  /// Optional [transactionId] could be used to cancel operation.
   Future<void> discoverAllServicesAndCharacteristics({String transactionId}) =>
       _manager.discoverAllServicesAndCharacteristics(
           this, transactionId ?? TransactionIdGenerator.getNextId());
 
+  /// List of discovered Services for the Device.
```

Comment:
```suggestion
  /// A list of discovered [Service]s for this peripheral.
```
I found in Flutter docs that it's ok to link to class and add `s` for plural.